SINGE_IO.cfg

```diff
@@ -0,0 +1,4 @@
+Data data1/X_SCODE_data.mat
+outdir Output
+num_replicates 2
```

Comment:
Should we call this file `SINGE_config.cfg`, `SINGE_options.cfg`, or `SINGE_settings.cfg`?  Most of it is I/O related, example the number of replicates.

Before merging, we'll need to update the readme to describe this file and how to run SINGE with it.  Is this space separated or whitespace-separated?  Tab-separated may be safest so we can support Windows file paths.